Apps/Src/ArrayConnection.c

```diff
@@ -0,0 +1,61 @@
+/* Copyright (c) 2020 UT Longhorn Racing Solar */
+
+#include "ArrayConnection.h"
+#include "Contactors.h"
+
+static void arrayStartup(OS_ERR *err) {
+    Precharge_Write(ARRAY_PRECHARGE, ON); // Turn on the array precharge
+
+    OSTimeDlyHMSM(0, 0, 5, 0, OS_OPT_TIME_HMSM_STRICT, &err); // wait 5 seconds
```

Comment:
Add this 5 seconds as a macro in the `config.h` file. After looking at the Tritium docs at Pickle, we'll likely have to change this. It'll just make it clearer what this number means.